src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IepLevelsResourceTest.kt

```diff
@@ -0,0 +1,70 @@
+package uk.gov.justice.digital.hmpps.incentivesapi.resource
+
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.incentivesapi.integration.IncentiveLevelResourceTestBase
+
+class IepLevelsResourceTest : IncentiveLevelResourceTestBase() {
```

Comment:
FYI: I've copied the old `IepReviewsResourceTest.kt` file and removed all the tests and code related to the `/iep/reviews/*` endpoints.